aws/fargate/manifest.yaml

```
@@ -12,22 +13,31 @@ inputs:
   container_port:
     type: number
     required: true
+    description: "Port number the container exposes (e.g., 8080)"
```

```suggestion
    description: "Port number the container exposes (e.g. 8080)"
```

I know `e.g.,` is valid, but it seems superfluous, can we update all the examples to remove the extra comma?